scripts/merge-dev.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+git checkout dev
+git merge main --no-ff --no-commit
+result=$?
+
+if [ $result = 1 ]; then
+   echo "There is a merge conflict. Aborting ..."
+   git merge --abort
+   exit 0
+fi
+git push dev
```

Comment:
We don't really care of handling the error because this is the GH runner's job. What I'll just do is:

```
#!/bin/bash

set -e

git checkout dev
git merge --no-ff --no-commit main
git push origin dev
```

Reply:
if we don't handle the error, we will get status failed for the GHA no ???  that's why i used `exit 0`

Reply:
indeed but I'd prefer a red build in order to get alerted, don't you?